packages/snaps-sdk/src/types/methods/dialog.ts

```diff
@@ -15,6 +17,10 @@ export enum DialogType {
   Prompt = 'prompt',
 }
 
+export type DefaultDialog = {
+  id: string;
```

Comment:
Should we support either `content` or `id` for the new API? Feels like a nice shorthand if you don't wanna update your interface.

Reply:
We can still keep the support for content yes.